kit/popups/src/Sheets/UIFullscreenSheet.tsx

```diff
@@ -1,23 +1,51 @@
 import * as React from 'react';
-import { StyleProp, StyleSheet, ViewStyle, StatusBar, useWindowDimensions } from 'react-native';
+import { StyleProp, StyleSheet, ViewStyle, useWindowDimensions } from 'react-native';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 import { UILayoutConstant } from '@tonlabs/uikit.layout';
+import { useAnimatedReaction } from 'react-native-reanimated';
 import { UISheet, UISheetProps } from './UISheet/UISheet';
 
+import { useShrinkContentUnderSheetContextProgress } from './ShrinkContentUnderSheet';
+import { useSheetProgress } from './UISheet/usePosition';
+
 export type UIFullscreenSheetProps = Omit<UISheetProps, 'countRubberBandDistance'> & {
     style?: StyleProp<ViewStyle>;
 };
 
+function MoveContentUnderSheet() {
+    const contentUnderSheetProgress = useShrinkContentUnderSheetContextProgress();
+    const positionProgress = useSheetProgress();
+
+    useAnimatedReaction(
+        () => {
+            // to a range 0-1 be starting from half of the progress to the end of it
+            return positionProgress.value * 2 - 1;
```

Comment:
Shouldn't it be like this? (to avoid negative values)
```suggestion
            return Math.max(0, positionProgress.value * 2 - 1);
```

Reply:
It's already restricted :) https://github.com/tonlabs/UIKit/pull/1475/files#diff-b1f12046ab68428094011905dacf501378fba2059da9e42d65ca0e967bc32446R169